tests/TestRunner/app/MethodCallsTests.js

```diff
@@ -181,6 +200,13 @@ describe(module.id, function () {
         var actual = TNSGetOutput();
         expect(actual).toBe("instance baseMethod called");
     });
+    it('Derived_OverloadedInstanceBaseMethod', function () {
+        var instance = TNSDerivedInterface.alloc().init();
+        instance.baseMethod(1);
+       
+        var actual = TNSGetOutput();
+        expect(actual).toBe("overloaded instance baseMethod: called");
```

Comment:
I don't see this method being overridden in `TNSDerivedInterface`, and the string that you are expecting is actually the one from `TNSBaseInterface `